project/config/afbini/config/views.view.publication_search.yml

```diff
@@ -105,8 +104,8 @@ display:
           total_pages: null
           id: 0
           tags:
-            next: 'Next ›'
-            previous: '‹ Previous'
+            next: ››
+            previous: ‹‹
```

Comment:
Was this a requested change? Should it not have the words?

Reply:
I think this is correct, next and previous should be blank and First and Last are to have the words